DatadogSessionReplay/Sources/SessionReplayConfiguration.swift

```diff
@@ -72,7 +79,7 @@ extension SessionReplay {
         ///   - customEndpoint: Custom server url for sending replay data. Default: `nil`.
         public init(
             replaySampleRate: Float,
-            textAndInputPrivacyLevel: SessionReplayTextAndInputPrivacyLevel,
+            textAndInputPrivacyLevel: TextAndInputPrivacyLevel,
```

Comment:
Open to discuss if we'd better have all 3 privacy levels required, or if we are fine with providing default values for 2 of them (we need at least one required parameter so than we can distinguish this initializer from the old one).

Reply:
/opinion

I would keep all levels without defaults and only add once the deprecated APIs are removed in major version.